src/util/KvAuth0.js

```diff
@@ -285,7 +326,11 @@ export default class KvAuth0 {
 		this[sessionPromise] = new Promise(resolve => {
 			// Ensure browser clock is correct before checking session
 			syncDate().then(() => this[initWebAuth]()).then(() => {
-				this.webAuth.checkSession({}, (err, result) => {
+				this.webAuth.checkSession({
+					audience: this.audience,
+					responseType: 'token id_token',
+					scope: this.scope,
+				}, (err, result) => {
```

Comment:
Will we also need to update CPS, how might this differ in that world where it's client side auth only?

Reply:
CPS does not have any MFA management pages or features, so there's nothing to update there. MFA has always client-side auth only, so nothing is changing there.